src/UniversalRewardsDistributor.sol

```diff
@@ -122,12 +122,16 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
             ErrorsLib.INVALID_PROOF_OR_EXPIRED
         );
 
+        require(claimable >= claimed[account][reward], ErrorsLib.ROOT_MISCONFIGURED);
+
         amount = claimable - claimed[account][reward];
 
         require(amount > 0, ErrorsLib.ALREADY_CLAIMED);
```

Comment:
```suggestion
        amount = claimable.zeroFloorSub(claimed[account][reward]);

        require(amount > 0, ErrorsLib.ALREADY_CLAIMED);
```

would that correspond to what you mean @QGarchery ?

Reply:
```suggestion
        require(claimable > claimed[account][reward], ErrorsLib.ALREADY_CLAIMED);

        amount = claimable - claimed[account][reward];
```
I had something like this in mind, potentially changing `ALREADY_CLAIMED` with something else, like `CLAIMABLE_TOO_LOW`

Reply:
Ah right, that works too! Isn't it better @julien-devatom ?

Reply:
I agree we're basically doing the check twice

Reply:
Go!